src/encoded/types/file.py

```diff
@@ -1119,19 +1123,35 @@ def is_file_to_download(properties, file_format, expected_filename=None):
         return filename
 
 
-@view_config(name='download', context=File, request_method='GET',
-             permission='view', subpath_segments=[0, 1])
+@view_config(name='download', context=File, request_method='GET', permission='view', subpath_segments=[0, 1])
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [download is too complex (32)](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4885443)